_data/blogs.yml

```diff
@@ -3,10 +3,10 @@
 # This is to remove conflicts with other people adding entries.
 
 # The Format is :
-# - url: URL to the blog
-#   title: Title of the blog
-#   img: Image of the blog
-#   author_name: Name of the author of the blog (most likely you)
+# - url: https://myblogs938167194.wordpress.com/2018/01/01/my-google-code-in-participation/
```

Comment:
You don't have to change these lines, you have to keep the structure, it was a syntax for showing how to add the blog.You have to use the same syntax for adding the blog.